src/views/certifications/views/landing/components/exam-card/types.ts

```diff
@@ -0,0 +1,10 @@
+import { CertificationProductSlug } from 'views/certifications/types'
+
+export interface ExamCardComingSoonProps {
```

Comment:
💭 Might be better to call this `ExamCardBaseProps` and then have `type ExamCardComingSoonProps = ExameCardBaseProps`, so that if we ever need to add more props to the 'coming soon' card it won't affect the generic `ExamCard`.

Reply:
Ah nice, I like it. Updated ✅